blocks/alert-bar-block/themes/commerce.json

```diff
@@ -29,15 +33,23 @@
 						"padding-top": "var(--global-spacing-5)",
 						"text-decoration": "none"
 					},
+					"link-hover": {
+						"color": "var(--global-neutral-8)"
+					},
 					"icon": {
-						"fill": "var(--global-white)"
+						"fill": "var(--global-white)",
+						"filter": "drop-shadow(0px 0px 0px rgba(0, 0, 0, 0.0))"
 					}
 				}
 			},
 			"desktop": {
 				"components": {
 					"link": {
-						"font-size": "var(--body-font-size)"
+						"font-size": "var(--body-font-size)",
+						"filter": "drop-shadow(0px 0px 0px rgba(0, 0, 0, 0.0))"
+					},
+					"link-hover": {
+						"color": "var(--global-neutral-8)"
```

Comment:
Same here... we should only override when necessary

Reply:
Made the changes